facebook/delphiFacebook/R/responses.R

```diff
@@ -31,7 +31,7 @@ load_responses_all <- function(params) {
 #'
 #' @importFrom stringi stri_split stri_extract stri_replace_all stri_replace
 #' @importFrom readr read_lines cols locale col_character col_integer
-#' @importFrom dplyr arrange desc case_when
+#' @importFrom dplyr arrange desc case_when mutate if_else
```

Comment:
don't need `if_else`? `ifelse` is in base

Reply:
Whoops, meant to use `if_else` there. Thanks for the catch.

Re: tests, yep.